src/map/battle.cpp

```diff
@@ -4108,12 +4108,19 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			break;
 		case SN_SHARPSHOOTING:
 		case MA_SHARPSHOOTING:
+			if (src->type == BL_MOB) { // TODO: Did these formulas change in the renewal balancing?
+				if (battle_config.skill_eightpath_algorithm == 1)
+					skillratio += 40 * skill_lv;
+				else
+					skillratio += 100 + 50 * skill_lv;
```

Comment:
I'm not sure this is right, I think @mrjnumber1 was saying the damage increases if the target is in an eight-path direction from the source.

If the source is at (0,0), a target at (4,4) will take extra damage, and a target at (7, 3) will take less damage.

Reply:
Good point! I didn’t read it like that!